### DISCUSSION TITLE
Notice: "Entity Framework Community Standup - Geographic Data with NetTopologySuite"

### DISCUSSION THREAD

**Discussion by FObermaier** [NetTopologySuite/NetTopologySuite#440] [Show and tell]
Watch Joe Amenta on NTS with Entity Framework:   
[Geographic Data with NetTopologySuite](https://www.youtube.com/watch?v=IHslY5rrxD0&feature=youtu.be&t=136)

Enjoy!

**Comment by DGuidi** [comment]
Thanks @airbreather , also for mentioning me, well appreciated! 
But I think you was too humble: most of the credits in the last years goes to you and @FObermaier. Thanks for your contribution :)
